app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -611,4 +614,11 @@ class StateFragmentPresenter @Inject constructor(
   }
 
   private fun isAudioShowing(): Boolean = viewModel.isAudioBarVisible.get()!!
+
+  fun updateSubmitButton(pendingAnswerError: String?) {
```

Comment:
This ought to have a brief KDoc explaining what it does (especially what a null vs. non-null error does).

Reply:
updated with KDoc   `/** Updates submit button UI as active if pendingAnswerError null else inactive. */`